doc/nrf/protocols/dect/index.rst

```diff
@@ -0,0 +1,144 @@
+.. _ug_dect:
+
+DECT NR+
+########
```

Comment:
Shouldn't the introduction page -> "Robust connectivity support" feature also have mention about DECT NR+ - https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/index.html ?

Reply:
yeah, we can add that